CRM/CiviAwards/Upgrader/Steps/Step1007.php

```diff
@@ -52,4 +53,12 @@ private function renameDbColumn() {
     CRM_Core_DAO::executeQuery($renameColumnSql);
   }
 
+  /**
+   * Sync log tables.
+   */
+  private function syncLogTables() {
+    $logging = new CRM_Logging_Schema();
+    $logging->fixSchemaDifferences();
```

Comment:
I think `$logging->fixSchemaDifferencesFor('civicrm_civiawards_award_detail')` would be a better option since this will only fix schema differences for the Award detail table. `$logging->fixSchemaDifferences()` will attempt to fix schema differences for all tables even though it won't be created for those that already have but will be more optimised if we create for exact table.

Reply:
yeah that is a good point, changed

Reply:
@tunbola can you review again please